Source/WebKit/WebProcess/Plugins/PDF/PDFPlugin.mm

```diff
@@ -539,12 +535,6 @@ - (PDFPage *)pageNearestPoint:(NSPoint)point currentPage:(PDFPage *)currentPage;
     m_pdfLayerController.get().delegate = m_pdfLayerControllerDelegate.get();
     m_pdfLayerController.get().parentLayer = m_contentLayer.get();
 
-    bool isFullFrame = isFullFramePlugin();
-    if (isFullFrame) {
-        // FIXME: <rdar://problem/75332948> get the background color from PDFKit instead of hardcoding it
```

Comment:
Should we keep the FIXME around, and move it to `PDFPluginBase::pluginBackgroundColor`?